Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx

```diff
@@ -3,19 +3,32 @@
 
 import React from 'react';
 import { FieldLabel, useAdaptiveFormContext } from '@bfc/adaptive-form';
-import { FieldProps, useRecognizerConfig, useShellApi } from '@bfc/extension-client';
+import { FieldProps, SDKKinds, useRecognizerConfig, useShellApi } from '@bfc/extension-client';
 import formatMessage from 'format-message';
 import { LuMetaData, LuType } from '@bfc/shared';
 
-const expectedResponsesPlaceholder = () =>
-  formatMessage(`> add some expected user responses:
-> - Please remind me to '{itemTitle=buy milk}'
-> - remind me to '{itemTitle}'
-> - add '{itemTitle}' to my todo list
->
-> entity definitions:
-> @ ml itemTitle
-`);
+const expectedResponsesPlaceholder = (id?: SDKKinds) => {
+  let placeholderText = '';
+  switch (id) {
+    case SDKKinds.CrossTrainedRecognizerSet:
+    case SDKKinds.LuisRecognizer:
+      placeholderText = formatMessage(`> add some expected user responses:
```

Comment:
@lei9444 these spaces you have added to the beginning of lines 16-22 (inclusive) will get rendered in monaco, please fix.

![Screen Shot 2021-07-08 at 11 06 51 AM](https://user-images.githubusercontent.com/1483492/124970305-9fb26580-dfdc-11eb-8f2f-2f119cda2bcc.png)